### DISCUSSION TITLE
Fancy Frontend

### DISCUSSION THREAD

**Discussion by Jemeni11** [kaf-lamed-beyt/ng-tech-events#21] [Ideas]
I was spreading the good news on WhatsApp when someone asked this:

![Screenshot_20230411-190144.png](https://user-images.githubusercontent.com/52603291/231250339-b9ae7e09-dfb1-4423-9a9e-151fa4f2fa82.png)

Why does it not have a fancy frontend?

This never occurred to me ngl 😅

It seems like a great idea though. 

Also seems like a good learning experience because I've never used docusaurus or MDX before

**Comment by kaf-lamed-beyt** [comment]
This occurred to me already, for a long time now 😃

I've not just been able to sit down and think of the technologies that we can use to make it happen. There's a design already sef. 

I was thinking of using Next.js before. But, now that you've mentioned markdown and docusaurus. I think it'll be worthy of giving it a shot. 

About the design... I may just have to reach out to the designer, so she can make some modifications.

**Reply by Jemeni11** [reply]
> This occurred to me already, for a long time now smiley

:skull: 
 
> I've not just been able to sit down and think of the technologies that we can use to make it happen. There's a design already sef.

Mad oh

> I was thinking of using Next.js before. But, now that you've mentioned markdown and docusaurus. I think it'll be worthy of giving it a shot.
> About the design... I may just have to reach out to the designer, so she can make some modifications.

Cool :+1:

**Comment by kaf-lamed-beyt** [comment]
On a second thought... let's go with Next.js. 

Since the site will be dynamic. People can search for the events, and we also add some SEO too.

**Reply by kaf-lamed-beyt** [reply]
Let's go with Chakra UI since we need to ship something fast. I'll create a branch that we can work on, in the afternoon. 

So what we're basically going to be doing is use GitHub's low level API via octokit to get the content in our Readme file and parse the content with packages like `markdown-it` or `remark` 

Let's see how it goes shaa... I'll update you. 😃

**Reply by Jemeni11** [reply]
Okay :+1:
